abusehelper/bots/archivebot/archivebot.py

```diff
@@ -85,11 +85,20 @@ def _unique_writable_file(directory, prefix, suffix):
         else:
             break
 
-    with os.fdopen(fd, "wb") as fileobj:
+    try:
+        fileobj = os.fdopen(fd, "wb")
```

Comment:
File descriptor ```fd``` is opened on line 79 without specifying ```os.O_BINARY``` (which is available only on Windows), however mode ```"wb"```is specified for ```fdopen()```. Not sure what is the end result of all this. This probably works just fine on UNIX systems, but it might do something weird on Windows.